### ISSUE TITLE
For checkboxes present under tasks section ‘Analyze survey results, Develop action plan……” Labels are not associated with check box control”: A11y_.NET 9 template_Home_Labels or Instructions.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#26938]
### GitHub Tags:
#A11yTCS; #A11ySev2; #Android; #WCAG3.3.2; #SH_.NET9template_Android_Dec2024; #Talkback; #.NET Core; #A11yMAS; #Labels or Instructions; #Device; #AINotInscope;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
Android Version: 14
Talkback Version:15.1

### Repro Steps:

1. Turn on Talkback and Install and open ".NET 9 template" app.
2. "Home Screen" will open
3. Swipe till "Task section" list items.
4. Observe that For checkboxes present under tasks section ‘Analyze survey results, Develop action plan……” Labels are not associated with check box control”.

### Actual Result:
For checkboxes present under tasks section ‘Analyze survey results, Develop action plan……” Labels are not associated with check box control”.

### Expected Result:
Analyze survey results, Develop action plan, Read a book list items Labels Talkback should be announced along with check box.

### User Impact:
Users who relay on screen reader will not get the proper information of those checkboxes.

### Attachment:
https://github.com/user-attachments/assets/a21adfa7-e7cc-4ab8-bf94-ecfca1744cc4

**Comment by sheiksyedm** [comment]
Duplicate #26897

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by jsuarezruiz** [comment]
Fixed by https://github.com/dotnet/maui/pull/26921

**Comment by keerthiduvvuri** [comment]
Verified in latest Environment, Issue is Fixed. Hence closing the bug.
Added attachment for reference.
#Regressed:03-06-25; #Closed; #A11yFixed;

[26938_Closing Attachment.zip](https://github.com/user-attachments/files/19102192/26938_Closing.Attachment.zip)
